staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -217,14 +221,16 @@ else {
   Write-Host "ctr.exe command not available"
 }
 
-# collect disk usage information
-if (Test-Path "C:\aks-tools\DU\du.exe") {
-  C:\aks-tools\DU\du.exe /accepteula
-  C:\aks-tools\DU\du.exe -l 1 C:\ProgramData\containerd\root\io.containerd.snapshotter.v1.windows\snapshots\ > "$ENV:TEMP\$timeStamp-du-snapshot-folder-size.txt"
-  $paths += "$ENV:TEMP\$timeStamp-du-snapshot-folder-size.txt"
+if ($args.Count -gt 0 -and $args -contains "disk-usage") {
```

Comment:
We have a command to collect disk-usage so the name may be confused. How about containers-disk-usage or snapshots-size? Could we not use contain since a valid name with different suffix can also pass the validation

Reply:
updated code to `switch` param